app/javascript/components/applications/application_form/HouseholdMemberForm.js

```diff
@@ -1,72 +1,72 @@
 import React from 'react'
 import AddressForm from './AddressForm'
-import { Field } from '~/utils/form/Field'
-import { MultiDateField } from '~/utils/form/MultiDateField'
 import { maxLengthMap } from '~/utils/formUtils'
 import formOptions from '~/components/applications/application_form/formOptions'
+import { FieldWrapper, SelectField } from '~/utils/form/final_form/Field'
+import { MultiDateField } from '~/utils/form/final_form/MultiDateField'
 
 const {
   relationshipOptions
 } = formOptions
 
-const HouseholdMemberForm = ({ i, formApi }) => {
+const HouseholdMemberForm = ({fieldName, index, form}) => {
```

Comment:
@software-project it's not clear here what type of information `fieldName` is. can we update the variable name or add a comment to further explain?

Reply:
@callaghanc what name do you find more descriptive? `formNamePrefix`, `fieldNamePrefix`? It's basically nested field name prefix. It's required for final for 'FieldArray'

Reply:
@software-project what values do we expect for `fieldName`? is it going to be "householdMember", or is it going to be the natural key?

Reply:
It's household member object name in the array, eg. `household_members[2]`

Reply:
ah ok thanks, so will the value of `fieldName` always be `` `household_members[#{index}]` ``?

Reply:
yes

Reply:
ok cool, then what do you think about passing ``fieldName={`household_members[${index}].middle_name`}`` to the fields in this form and removing the `fieldName` prop from `HouseholdMemberForm` entirely? that makes one less thing to think about here, since `fieldName` isn't really bringing any info that isn't already in `index`

Reply:
that would work, sure.

Reply:
cool i'll push that change